invoicing_app/templates/declaration_template.html

```diff
@@ -0,0 +1,9 @@
+<html>
+	<body>
+		<h1>Declared Tax Receipts</h1>
+		<p>Here is the report from period the {{ period }} :</p>
+		<p>Country: {{ country }}</p>
+        <p>DECLARED GTF: {{ declared_gtf }}</p>
+		<p>Thanks!</p>
+	</body>
+</html>
```

Comment:
here you have to add a space at the end of the file

Reply:
Done!